### PR TITLE
Fix pubsig process

### PR DESCRIPTION
**Pull request by hyeonLewis**
Import updated pub-signals process.

### CODE REVIEW

File: src/zkauth-circuit.ts [hunk 1]
```diff
@@ -25,14 +25,13 @@ export const ZkauthJwtV02 = {
 
         const payOff = header.length + 1; // position in base64-encoded JWT
         const payLen = payload.length;
-        const pay = base64url.decode(payload);
+        // toString('ascii') may result in some unicode characters to be misinterpreted
```
Comment (ian0371) [inline src/zkauth-circuit.ts:28]: @2dvorak  Why use `toString("ascii")` if it can be misinterpreted?

Comment (2dvorak) [reply src/zkauth-circuit.ts:28]: @ian0371 The `pay` variable is only used to parse claim **offsets**, and their exact values are not our concern.

**Review by ian0371 — APPROVED** [review]

File: src/zkauth-circuit.ts (hunk 1 above)
Comment (2dvorak) [reply src/zkauth-circuit.ts:28]: @ian0371 On second thought, the offsets could be wrong with a extreme coincidence where the ascii decoded characters includes `"`, `,` or `}`.

<details>
<summary>quick, unverified idea</summary>
For example, with below code:

```
let msg = "안녕";
let msg2 = Buffer.from(msg).toString('ascii');

console.log(Buffer.from(msg));
console.log(msg2);
console.log(Buffer.from(msg2));
```

I could see "안녕", which is `0xec 0x95 0x88 0xeb 0x85 0x95` in utf-8 encoding, includes `0x6c` and `0x6b` when decoded as ASCII, which corresponds to "l" and "k".

```
$ node index.ts
<Buffer ec 95 88 eb 85 95>
k
<Buffer 6c 15 08 6b 05 15>
```

So if a claim value includes one or two utf-8 characters that could be ASCII-decoded into something like `0x22 0x20 0x2c` (`" ,`), our regex may miscalculate the offset or length of that claim.
</details>

I expect that the claims of our concern (`sub`, `iss`, `aud`, `iat`, `exp`, `nonce`) would not include any unicode characters.

File: src/circuit-helpers.ts [hunk 2]
```diff
@@ -1,7 +1,46 @@
...
+
+export function toBase64(buffer: Buffer): string {
+    return base64url(buffer);
+}
+
+export function toHex(buffer: Buffer): string {
+    return buffer.toString("hex");
+}
+
 function bufferToUints(buffer: Buffer, chunkLen: number): string[] {
```
Comment (2dvorak) [inline src/circuit-helpers.ts:44]: Do we still need `bufferToUints`?

File: src/zkauth-circuit.ts (hunk 1 above)
Comment (ian0371) [reply src/zkauth-circuit.ts:28]: I agree that it won't be an immediate concern.
For future reference, please do note that a domain name can include unicode characters, thus affecting `iss` or `aud`.

File: src/circuit-helpers.ts (hunk 2 above)
Comment (hyeonLewis) [reply src/circuit-helpers.ts:44]: It's used by authBuilder.

Comment (hyeonLewis) [reply src/circuit-helpers.ts:44]: Oh, I think we can use just use `toUints`. Is it right?

Comment (2dvorak) [reply src/circuit-helpers.ts:44]: Yep, my intention was to replace all helper functions with proper combination of `fromXXX` and `toXXX` plus some SHA256 helpers (`sha256Pad`).

Comment (hyeonLewis) [reply src/circuit-helpers.ts]: @2dvorak Applied at [3f4ad1e](https://github.com/klaytn/zkauth-sdk/pull/6/commits/3f4ad1ea78ecd7c8ef3fab61674c92aea5d7ee43).

**Review by 2dvorak — APPROVED** [review]

**Pull request merged**